prisma/schema.prisma

```diff
@@ -773,6 +773,15 @@ model Log {
   @@index([createdAt, name], map: "Log.name_index")
 }
 
+model AuditLog {
+  id        Int @id @default(autoincrement())
+  createdAt DateTime @default(now()) @map("created_at")
+  table     String
+  message   String?
+  old       Json
+  new       Json
+}
+
```

Comment:
I think making this polymorphic denormalizing a row for each thing we're auditing would be valuable. See `Wallet` and `WalletLND` etc.

Reply:
Done in 69a8bdee